client/src/components/user-admin/EditUsers.jsx

```diff
@@ -9,6 +9,9 @@ const EditUsers = ({ userToEdit, backToSearch, updateUserDb, projects, updateUse
   const [isActive, setIsActive] = useState(userToEdit.isActive);
   const [isAdmin, setIsAdmin] = useState(userToEdit.accessLevel === "admin");
 
+  // Function to check if the current user is the super admin
```

Comment:
Semantics, but this comment says you provided a function but you've provided a boolean

Reply:
Good catch trillium! I changed approach mid way but forgot to change the comment. Addressing it right away